everything_shell/environ.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+/**
+ * main - prints environment variable using global variable environ
+ *
+ * Return: 0 success.
+ */
+int main()
+{
+	unsigned int i;
+	extern char **environ;
```

Comment:
Extern makes use of <unistd.h> but is not included here. Due to implicit declaration, the code may work, but it is good practice to include it.

Reply:
Okay, this makes sense